config.rb

```diff
@@ -34,7 +34,10 @@ def attempts_api_path
 
       # @return [OpenSSL::PKey::RSA]
       def attempts_private_key
-        @attempts_private_key ||= OpenSSL::PKey::RSA.new(@config.fetch('irs_private_key')) if @config.fetch('irs_private_key').present?
+        irs_private_key = @config.fetch('irs_private_key')
+        return nil unless irs_private_key.present?
+        irs_private_key = irs_private_key.gsub('\n', "\n")
```

Comment:
is this substituting an unescaped literal character `\n` for a real newline? I think that means we're likely setting the value incorrectly

Reply:
it is working fine in my local without gsub, but when I add a multiline private key in cloud gov it gives no start line error I tried different ways to add a key but never worked then I converted multiline to single line did gsub, if you know, and help me how to add multiline key in cloud.gov environment variable, it will be helpful

Reply:
in PEM format, I think newlines are not significant, so we could just upload as a single line?

Reply:
![image](https://user-images.githubusercontent.com/109746710/198335697-af35a4ea-2515-4d9a-bbd4-bcf49b71b6f0.png)
it only works if I add like this(and gsub)

Reply:
https://stackoverflow.com/a/43097899

Reply:
Dang, thanks for those links, what a bummer